instrumentation/servlet/servlet-rw/src/test/java/io/opentelemetry/javaagent/instrumentation/hypertrace/servlet/rw/reader/BufferedReaderInstrumentationTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright The Hypertrace Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.javaagent.instrumentation.hypertrace.servlet.rw.reader;
+
+import io.opentelemetry.api.trace.Span;
+import java.io.BufferedReader;
+import java.io.CharArrayReader;
+import java.io.IOException;
+import org.ServletReadWriteContextAccess;
+import org.TestBufferedReader;
+import org.hypertrace.agent.core.instrumentation.buffer.*;
+import org.hypertrace.agent.testing.AbstractInstrumenterTest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class BufferedReaderInstrumentationTest extends AbstractInstrumenterTest {
```

Comment:
Could you please add tests for all methods that read data on the BufferedReader? The same for `PrintWriter`